corgidrp/data.py

```diff
@@ -354,9 +353,82 @@ def __init__(self, data_or_filepath, pri_hdr=None, ext_hdr=None, input_dataset=N
         if 'DATATYPE' not in self.ext_hdr or self.ext_hdr['DATATYPE'] != 'Dark':
             raise ValueError("File that was loaded was not a Dark file.")
 
+class NonLinearityCalibration(Image):
```

Comment:
Can we call this NonLinearityCorrection() instead?

Reply:
We could, but why? I personally think of _correcting_ as an action and a _calibration_ file/data as the information (it's true there's also the action of performing a calibration).

Reply:
Fair. We can leave it as is.